README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/brj/fish-cookbook) - Tips and recipes for fish-shell users
+- [fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup fish Shell with fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
@ellerbrock I am far from anything like that. I did make fisherman, the plugin manager and a few other little fish projects. You are encouraged to join the Gitter room like @laughedelic recommends and also the fisherman [Slack](http://join.fisherman.sh) room :)